lib/parse.js

```diff
@@ -21,6 +22,7 @@ function parse(css, options) {
     var needInfo = options && options.needInfo || Defaults.NEED_INFO;
     var rule = options && options.rule ||
         (syntax === 'js' ? Defaults.JS_RULE : Defaults.CSS_RULE);
+    var tabSize = options && options.tabSize || Defaults.TAB_SIZE;
```

Comment:
I think there should be some kind of a check that `tabSize` is a number and is greater than 0.
(Is there a case when you'd want `tabSize` to equal 0?)